ui-core/src/participant/landing/ConfiguredButton.tsx

```diff
@@ -2,7 +2,7 @@ import classNames from 'classnames'
 import React from 'react'
 import { Link } from 'react-router-dom'
 
-import { requireOptionalString, requirePlainObject, requireString } from 'util/validationUtils'
+import { requireOptionalString, requirePlainObject, requireString } from '../../participant/util/validationUtils'
```

Comment:
These paths seem unnecessarily long. Couldn't this be `../util/validationUtils`?

Reply:
probably leftover from moving them around.